homeassistant/components/flipr/__init__.py

```diff
@@ -0,0 +1,90 @@
+"""The Flipr integration."""
+from datetime import timedelta
+import logging
+
+from flipr_api import FliprAPIRestClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import CONF_FLIPR_ID, DOMAIN, MANUFACTURER, NAME
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Flipr from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    coordinator = FliprDataUpdateCoordinator(hass, entry)
+    await coordinator.async_config_entry_first_refresh()
+    hass.data[DOMAIN][entry.entry_id] = coordinator
+
+    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
```

Comment:
Missing return value typing. It should return a boolean. Please type the whole signature when adding type annotations to a function.